addons/cookoff/CfgEden.hpp

```diff
@@ -11,7 +11,7 @@ class Cfg3DEN {
                         tooltip = CSTRING(enable_tooltip);
                         expression = QUOTE(if !(_value) then {_this setVariable [ARR_3('%s',_value,true)];};);
                         typeName = "BOOL";
-                        condition = "objectVehicle";
+                        condition = "objectVehicle + objectHasInventoryCargo";
```

Comment:
Yeah I wasn't quite sure if you can get vehicles without inventory space and figured it was safer to just encompass all this way.